testing/travis-fork-check.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import logging
```

Comment:
This this script is indented to be small, that makes complete sense
> On May 11, 2016, at 12:44, Brian Hicks <notifications@github.com> wrote:
> 
> In testing/travis-fork-check.py <https://github.com/CiscoCloud/mantl/pull/1415#discussion_r62892992>:
> 
> > @@ -0,0 +1,30 @@
> > +#!/usr/bin/env python
> > +from __future__ import print_function
> > +import logging
> are you using logging because it prints to stderr? It might be simpler to just sys.stderr.write('blah\n')
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/CiscoCloud/mantl/pull/1415/files/dce2ccb628c69104764c0cad923e016abacadee1#r62892992>